#166272323 integrate travis ci with readme badge

emmygozi opened the pull request:
#### What does this PR do?

This Pull Request integrates Travis Continuous Integration with project and adds 
build badge to project readme

#### Description of Task to be completed?

- Add project github repo to Travis for Continuous Integration

- Create `.travis.yml` file with project configurations

- Add build badge to project Readme.md file

#### How should this be manually tested?

- `git clone https://github.com/andela/kingsmen-ah-backend.git` 

- `npm install` project dependencies

- Push project to github and click on Travis build

- Travis build fails beacause there is no test provided yet


#### Any background context you want to provide?

Discovered when working in a team, one has to be given admin privileges 
to add project repo to Travis CI 
#### What are the relevant pivotal tracker stories?

https://www.pivotaltracker.com/story/show/166272323

#### Screenshots (if appropriate)

N/A
### Questions:

N/A

Commit:
chore-[166272323]: integrate travis ci with readme badge

- add repo to travis ci for continuous integration
- create .travis.yml file with project configurations
- add build badge to project readme

[Finishes #166272323]

Commit:
chore-[166272323]: update readme badge

- add build badge to project readme

[Finishes #166272323]

Commit:
Merge branch 'develop' of https://github.com/andela/kingsmen-ah-backend into 166272323-ch-integrate-travisci-badge

emmygozi committed:
chore-[166272323]: save progress before update

- save work progress before pulling develop branch

[Finishes #166272323]

emmygozi committed:
chore-[#166272323]: integrate travis with slack

- integrate travis with slack to send notifications
- encrypt notifications with gem and travis CLI

[Finishes #166272323]

emmygozi committed:
Merge branch 'develop' of https://github.com/andela/kingsmen-ah-backend into 166272323-ch-integrate-travisci-badge

fidelisojeah reviewed (changes requested):
fix issues reported by hound

can you confirm your travis sends updates to coveralls ?

emmygozi committed:
chore-[#166272323]: confirm travis sends to coveralls

- install coveralls in package.json
fix hound ci error

[Finishes #166272323]

emmygozi committed:
chore-[#166272323]: update test configuration

- update package.json with nyc configuarations for unit test

[Finishes #166272323]
